customTests/azure_gpu_bandwidth.nhc

```diff
@@ -173,8 +169,187 @@ function check_gpu_bw()
         result_lines_array["$current_test"]="$extracted_lines"
     fi
 
-    evaluate_result $EXP_CUDA_BW $EXP_P2P_BW
+    evaluate_nvBW_result $EXP_CUDA_BW $EXP_P2P_BW
  
     return 0
 }
 
+function get_OAM_pairs(){
+    # Get OAM pairs
+    # return array where the index is the GPU id and the value is the OAM pair
+    oam_check=$(rocm-smi --showserial | grep GPU)
+
+    if [[ "$oam_check" == *"Not supported"* ]]; then
+        echo ""
```

Comment:
Need a message here?

Reply:
Embarrassing. Yea I will. Thanks!

Reply:
Actually, I forgot I meant to do this. Let me add a comment. Essentially if OAM is not supported I just want to echo an empty return value.